.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -569,6 +569,30 @@ public static float ParseNumberFromDurationTimex(string timex)
             return float.Parse(numberStr);
         }
 
+        public static int ParseHourFromTimeTimex(string timex)
+        {
+            var start = timex.IndexOf(Constants.TimeTimexPrefix) + 1;
+            var end = timex.IndexOf(Constants.TimeTimexConnector);
+            end = end > 0 ? end : timex.Length;
+            var hourStr = timex.Substring(start, end - start);
```

Comment:
From now on, let's try to write such functions using Spans, to avoid creating a lot of small string objects garbage collected all the time. Please open an issue to refactor existing Timex parsing code to do the same.

Reply:
@aitelint Please ACK this comment. Thanks.

Reply:
@tellarin, issue created

Reply:
@tellarin, the problem is that net462 does not support parsing of spans, so we would need to convert the span back to string when passing it to int.TryParse